src/typeahead/typeahead.directive.ts

```diff
@@ -159,13 +162,6 @@ export class TypeaheadDirective implements OnInit, OnDestroy {
       e.preventDefault();
       return;
     }
-
-    // if tab default browser behavior will select next input field, and
-    // therefore we should close the items list
-    if (e.keyCode === 9) {
```

Comment:
Also, the only thing this line does is hides the results. This happens when the user blurs the component as well.